app/models/journal_manager.rb

```diff
@@ -254,4 +259,15 @@ def self.create_custom_field_data(journable, journal)
       journal.customizable_journals.build custom_field_id: cv.custom_field_id, value: cv.value
     end
   end
+
+  # Wrap send_notification so it defaults to true, when it's nil
+  def send_notification
+    return true if @send_notification.nil?
+    @send_notification
+  end
+
+  # Need to clear the notification setting after each usage otherwise it might be cached
+  def self.clear_notification
```

Comment:
I'd also vote for `reset_notification`, because clear sounds like setting it to false...